index.html

```diff
@@ -735,6 +735,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="https://wiki.freebsd.org/PhilipPaeps"><i class="icon fa fa-external-link"></i></a>
+                      <a target="default" href="https://github.com/ppaeps"><i class="icon social_github"></i></a>
```

Comment:
23 September 2017

Reply:
It is already intended as it shows on my repo